Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml

```diff
@@ -596,8 +641,8 @@ tasks:
     view: |-
       {
         "position": {
-          "x": -400,
-          "y": 1160
+          "x": -410,
+          "y": 1350
         }
       }
     note: false
```

Comment:
Adding comment for Line 606, should we select the option for "Skip this branch if this automation/playbook is unavailable". I don't see this selected.

Reply:
Line 651 is related to task 21 which is calling subplaybook "Cortex ASM - Decision."  Because this is part of the ASM pack, I don't think we need that option selected because there should never be a situation where "Cortex ASM - Remediation Path Rules" subplaybook is installed but "Cortex ASM - Decision" is not.  Make sense?